cmd/sgbouncewizard/main.go

```diff
@@ -55,12 +136,125 @@ func ListRules(w http.ResponseWriter, r *http.Request) {
 
 func generateHash(pwd []byte) string {
 	hash, err := bcrypt.GenerateFromPassword(pwd, bcrypt.MinCost)
+	if err != nil {
+		log.Println(err)
+		return "error"
+	}
+	return string(hash)
+}
+
+func createRuleDB(rule *BounceRule) []byte {
+	db, err := sql.Open("mysql", "root:root@tcp(127.0.0.1:3306)/drop_rules")
```

Comment:
The connection to the db should only be opened once, and passed around. Go is really good and managing a connection pool to the db, however, if we open and close the db every time we make a query, we lose the benefits of the connection pool.